tpm2_pytss/ESAPI.py

```diff
@@ -823,7 +823,7 @@ def load(
 
     def load_external(
         self,
-        in_private: TPM2B_SENSITIVE,
+        in_sensitive: TPM2B_SENSITIVE,
         in_public: TPM2B_PUBLIC,
```

Comment:
the argument to Esys_LoadExternal / TPM2_LoadExternal is called inPrivate, so it should be left as in_private